### PR TITLE
Restricted visible tabs for admin and manager

### LINKED ISSUE #385: Change tab displayment under different roles
For now we get forbidden for orders and products list on admin dashboard
Under admin role we should see users tab
Under manager role we should see only products and orders tab

image

### PR DESCRIPTION
**Pull request by mhevyk**
https://github.com/user-attachments/assets/4f1f06f6-9659-4d0f-82ba-e40b38b9da12

image

image

### CODE REVIEW

File: code/src/layouts/dashboard-layout/DashboardLayout.tsx [hunk 1]
```diff
@@ -1,37 +1,23 @@
...
+    return <DashboardAdminLayout />;
+  }
 
-const DashboardLayout = () => {
-  const tabLabels = dashboardTabs.map((tab) => (
-    <DashboardTab key={tab.name} tab={tab} />
-  ));
+  if (userRole === ROLES.SHOP_MANAGER) {
+    return <DashboardManagerLayout />;
+  }
```
Comment (D1nviz) [inline code/src/layouts/dashboard-layout/DashboardLayout.tsx]: I guess you can use object here and use layouts by Role as a key

Comment (dmtrth25) [inline code/src/layouts/dashboard-layout/DashboardLayout.tsx]: What do you think to add `switch case` here?
I think that in future we can simpler to add some new roles

Comment (mhevyk) [reply code/src/layouts/dashboard-layout/DashboardLayout.tsx]: Thanks! I used `switch case`

Comment (mhevyk) [reply code/src/layouts/dashboard-layout/DashboardLayout.tsx]: Thanks! I used `switch case`! I discussed it with you

**Review by D1nviz — APPROVED** [review]

**Review by dmtrth25 — APPROVED** [review]

**Review by OliaHrytsak — APPROVED** [review]

**Review by heshvab — APPROVED** [review]

**Pull request merged**